{
  "uid" : "cfbe78d88a3df731",
  "name" : "Pay with PayPal",
  "fullName" : "Paid blocks.Pay with PayPal",
  "historyId" : "e0ca1dae1232c8da376862907c199a8b",
  "time" : {
    "start" : 1595622118493,
    "stop" : 1595622189282,
    "duration" : 70789
  },
  "description" : "",
  "descriptionHtml" : "",
  "status" : "failed",
  "statusDetails" : {
    "message" : "Error: Command failed: yarn wp-env run tests-wordpress cat wp-content/debug.log\n                ℹ Starting 'cat wp-content/debug.log' on the tests-wordpress container. \n                \n                Starting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... \n                \nStarting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... done\ncat: wp-content/debug.log: No such file or directory\n                ✖ Command failed with exit code 1\n                Command failed with exit code 1\n                error Command failed with exit code 1.\n                ",
    "trace" : "Error: Command failed: yarn wp-env run tests-wordpress cat wp-content/debug.log\n                ℹ Starting 'cat wp-content/debug.log' on the tests-wordpress container. \n                \n                Starting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... \n                \nStarting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... done\ncat: wp-content/debug.log: No such file or directory\n                ✖ Command failed with exit code 1\n                Command failed with exit code 1\n                error Command failed with exit code 1.\n                \n                    at checkExecSyncError (child_process.js:630:11)\n                    at execSync (child_process.js:666:15)\n                    at execSyncShellCommand (/home/travis/build/Automattic/jetpack/tests/e2e/lib/utils-helper.js:27:9)\n                    at _callee10$ (/home/travis/build/Automattic/jetpack/tests/e2e/lib/page-helper.js:209:9)\n                    at tryCatch (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:45:40)\n                    at Generator.invoke [as _invoke] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:274:22)\n                    at Generator.prototype.<computed> [as next] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:97:21)\n                    at asyncGeneratorStep (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)\n                    at _next (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)\n                    at /home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:32:7",
    "flaky" : false
  },
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusDetails" : {
      "message" : "Error: Command failed: yarn wp-env run tests-wordpress cat wp-content/debug.log\n                ℹ Starting 'cat wp-content/debug.log' on the tests-wordpress container. \n                \n                Starting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... \n                \nStarting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... done\ncat: wp-content/debug.log: No such file or directory\n                ✖ Command failed with exit code 1\n                Command failed with exit code 1\n                error Command failed with exit code 1.\n                ",
      "trace" : "Error: Command failed: yarn wp-env run tests-wordpress cat wp-content/debug.log\n                ℹ Starting 'cat wp-content/debug.log' on the tests-wordpress container. \n                \n                Starting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... \n                \nStarting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... done\ncat: wp-content/debug.log: No such file or directory\n                ✖ Command failed with exit code 1\n                Command failed with exit code 1\n                error Command failed with exit code 1.\n                \n                    at checkExecSyncError (child_process.js:630:11)\n                    at execSync (child_process.js:666:15)\n                    at execSyncShellCommand (/home/travis/build/Automattic/jetpack/tests/e2e/lib/utils-helper.js:27:9)\n                    at _callee10$ (/home/travis/build/Automattic/jetpack/tests/e2e/lib/page-helper.js:209:9)\n                    at tryCatch (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:45:40)\n                    at Generator.invoke [as _invoke] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:274:22)\n                    at Generator.prototype.<computed> [as next] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:97:21)\n                    at asyncGeneratorStep (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)\n                    at _next (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)\n                    at /home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:32:7",
      "flaky" : false
    },
    "steps" : [ {
      "name" : "Can visit the block editor and add a Pay with PayPal block",
      "time" : {
        "start" : 1595622118494,
        "stop" : 0,
        "duration" : -1595622118494
      },
      "status" : "unknown",
      "statusDetails" : {
        "flaky" : false
      },
      "steps" : [ ],
      "attachments" : [ {
        "uid" : "2fd37b05354128fa",
        "name" : "Test failed: Paid blocks :: Pay with PayPal",
        "source" : "2fd37b05354128fa.png",
        "type" : "image/png",
        "size" : 45080
      } ],
      "parameters" : [ ],
      "stepsCount" : 0,
      "attachmentsCount" : 1,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "Paid blocks"
  }, {
    "name" : "testClass",
    "value" : "Paid blocks"
  }, {
    "name" : "testMethod",
    "value" : "Pay with PayPal"
  }, {
    "name" : "package",
    "value" : "Paid blocks"
  }, {
    "name" : "resultFormat",
    "value" : "allure1"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : true,
  "extra" : {
    "tags" : [ ]
  },
  "source" : "cfbe78d88a3df731.json",
  "statusDetailsSafe" : {
    "message" : "Error: Command failed: yarn wp-env run tests-wordpress cat wp-content/debug.log\n                ℹ Starting 'cat wp-content/debug.log' on the tests-wordpress container. \n                \n                Starting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... \n                \nStarting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... done\ncat: wp-content/debug.log: No such file or directory\n                ✖ Command failed with exit code 1\n                Command failed with exit code 1\n                error Command failed with exit code 1.\n                ",
    "trace" : "Error: Command failed: yarn wp-env run tests-wordpress cat wp-content/debug.log\n                ℹ Starting 'cat wp-content/debug.log' on the tests-wordpress container. \n                \n                Starting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... \n                \nStarting 5c5d92595c30b14aa520446a7eafd734_mysql_1_a3359e0d9f67 ... done\ncat: wp-content/debug.log: No such file or directory\n                ✖ Command failed with exit code 1\n                Command failed with exit code 1\n                error Command failed with exit code 1.\n                \n                    at checkExecSyncError (child_process.js:630:11)\n                    at execSync (child_process.js:666:15)\n                    at execSyncShellCommand (/home/travis/build/Automattic/jetpack/tests/e2e/lib/utils-helper.js:27:9)\n                    at _callee10$ (/home/travis/build/Automattic/jetpack/tests/e2e/lib/page-helper.js:209:9)\n                    at tryCatch (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:45:40)\n                    at Generator.invoke [as _invoke] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:274:22)\n                    at Generator.prototype.<computed> [as next] (/home/travis/build/Automattic/jetpack/node_modules/regenerator-runtime/runtime.js:97:21)\n                    at asyncGeneratorStep (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:3:24)\n                    at _next (/home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:25:9)\n                    at /home/travis/build/Automattic/jetpack/node_modules/@babel/runtime/helpers/asyncToGenerator.js:32:7",
    "flaky" : false
  },
  "statusMessage" : {
    "empty" : false,
    "present" : true
  },
  "parameterValues" : [ ]
}